integrations/utils.ts

```diff
@@ -448,6 +425,9 @@ export function test(
 test.only = (name: string, config: TestConfig, testCallback: TestCallback) => {
   return test(name, config, testCallback, { only: true })
 }
+test.skip = (name: string, config: TestConfig, testCallback: TestCallback) => {
+  return test(name, config, testCallback, { skip: true })
+}
```

Comment:
Introduced this during debugging and while this is not used right now, I think this is still useful (similar to how `.only` and `.debug` is not used right now).